content/en/docs/setup/getting-started/index.md

```diff
@@ -45,7 +45,7 @@ Follow these steps to get started with Istio:
     You can pass variables on the command line to download a specific version
     or to override the processor architecture.
     For example, to download Istio 1.6.8 for the x86_64 architecture,
-    run `curl -L https://istio.io/downloadIstio | ISTIO_VERSION=1.6.8 TARGET_ARCH=x86_64 sh -`.
+    run `curl -L https://istio.io/downloadIstio | ISTIO_VERSION=1.6.8 TARGET_ARCH=x86_64 sh -`
```

Comment:
Technically speaking, the `.` is correctly ending the sentence. Maybe it would be better to change it to:
```suggestion
    run:

    {{< text bash >}}
    curl -L https://istio.io/downloadIstio | ISTIO_VERSION=1.6.8 TARGET_ARCH=x86_64 sh -
    {{< /text >}}

```